notebooks/official/workbench/fraud_detection/fraud-detection-model.ipynb

```diff
@@ -9,6 +9,57 @@
         "# Build a fraud detection model on Vertex AI"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        serving_container_image_uri="us-docker.pkg.dev/vertex-ai/prediction/sklearn-cpu.0-24:latest",
</pre><p>Can we pull this out as a constant variable? Also, let's use the latest version 1.0 listed here: https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>